README.md

```diff
@@ -1,4 +1,9 @@
-<img src="https://github.com/neuropoly/axondeepseg/blob/master/docs/source/_static/logo_ads-alpha.png" width="385">
+
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/axondeepseg/doc-figures/blob/main/logo/logo_ads-dark-alpha.png?raw=true" width="385">
+  <img alt="" src=https://github.com/axondeepseg/doc-figures/blob/main/logo/logo_ads-alpha.png?raw=true" width="385">
+</picture>
```

Comment:
This syntax LGTM! The [Art Direction](https://developer.mozilla.org/en-US/docs/Web/HTML/Responsive_images#art_direction) section of the Responsive Images guide explicitly mentions this syntax, and also says that it's fine to remove the "light" case due to the <img> fallback case. :)

Reply:
Thanks!!